tests/console/start_systemd_testkit_offline.pm

```diff
@@ -92,24 +92,14 @@ sub parse_results_from_output {
                 #   $openQA_result = $self->record_testresult('softfail');
                 #   $softfail_result = 119565;
                 #} else {
-                my ($openQA_result, $softfail_result);
-                if ($error_line =~ /invalid version.*expected.*/ && is_sle('=15-SP5')) {
-                    $openQA_result = $self->record_testresult('softfail');
-                    $softfail_result = 1203060;
-                }
-                else {
-                    my $openQA_result = $self->record_testresult('fail');
-                    #$softfail_result = 0;
-                    #}
-                    $softfail_result = 0;
-                }
+                my $openQA_result = $self->record_testresult('fail');
+                #$softfail_result = 0;
```

Comment:
This was not used before and also now. Why not remove it ?

Reply:
@dzedro 
Sorry, I can't understand your comment.

The code has some annotation now to tell us how to add a softfail 
`            if ($outcome eq 'failed') {
                # In case you need to add soft failure, use the following commented code as a guide
                #my ($openQA_result, $softfail_result);
                #if ($error_line =~ /problems.*hostagent.service/) {
                #   $openQA_result = $self->record_testresult('softfail');
                #   $softfail_result = 119565;
                #} else {
                my $openQA_result = $self->record_testresult('fail');
                #$softfail_result = 0;
                #}
`

I just revert the code I did on via PR https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/15492 and revised the code via perl tidy check.

Can you please share more lights?

Reply:
It's just about the commented line, it can be removed.

Reply:
Thank you, then I will keep the lines for now.